XLS-0034-paychan-escrow-for-tokens/README.md

```diff
@@ -1,10 +1,12 @@
-<pre class="notranslate">
-Title:       <b>Token-Enabled Escrows and Payment Channels</b>
-Type:        Draft
-PR:          <a href="https://github.com/XRPLF/rippled/pull/4396">4396</a>
-Author:      Richard Holland, Xaman <a href="https://github.com/RichardAH/">(RichardAH)</a>
-             Denis Angell, Transia <a href="https://github.com/dangell7/">(dangell7)</a>
-Affiliation: <a href="https://xrplf.org" rel="nofollow">XRPLF</a>
+<pre>
+xls: 34
+title: Token-Enabled Escrows and Payment Channels
+description: Amendment to enable Escrows and PayChannels to use Trustline balances in addition to XRP
+author: Brandon Wilson <bwilson@ripple.com>
```

Comment:
Wrong authors - I'm seeing a pattern here, so if there are authors specified in the previous diff, the new version should use the same list of authors